### DISCUSSION TITLE
Welcome to CiderPress II Discussions!

### DISCUSSION THREAD

**Discussion by fadden** [fadden/CiderPress2#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

(I haven't used github "discussions" before. Let's give it a try.)

**Comment by 1st-elektron** [comment]
Thank you for developing Ciderpress II.
It is very usefull for me
